haskell-xmpp.cabal

```diff
@@ -37,7 +37,7 @@ flag debug
 library
   default-language: Haskell2010
   Hs-Source-Dirs: ./src
-  Build-Depends: base > 4.13.0.0 && <= 4.14.0.0
```

Comment:
4.14 is for `8.10`, are you sure that the project works with it?
Also, could we build with the three latest major versions on CI, do you think that it would be possible with herculesci?

I also think that we could just add GitHub Actions for that, if that it troublesome for the current CI 🙂

Reply:
>  Also, could we build with the three latest major versions on CI, do you think that it would be possible with herculesci?

Of course!
We have these ones supported on the latest nixpkgs: https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/haskell-packages.nix#L46

But we can make additional pins to support basically every version since the inception of nix. 

> I also think that we could just add GitHub Actions for that, if that it troublesome for the current CI 

I don't understand the appeal of github actions. I can't even boot my laptop with it.

Reply:
I am not a nix user, so can't help with your issues, but I am not sure I understand how you supposed to use that on your laptop if it is the CI 🤔

Reply:
joke, nevermind.